### DISCUSSION TITLE
RuntimeError("Distributed package doesn't have NCCL " "built in")

### DISCUSSION THREAD

**Discussion by Blacksingh** [voicepaw/so-vits-svc-fork#377] [Q&A] [answered]
Everything goes fine for me until I try to train
Then I run into these issue

C:\Users\jerme\anaconda3\envs\so-vitts-fork\python.exe: Error while finding module specification for '__main__' (ValueError: __main__.__spec__ is None)

[W ..\torch\csrc\distributed\c10d\socket.cpp:601] [c10d] The client socket has failed to connect to [kubernetes.docker.internal]:49465 (system error: 10049 - The requested address is not valid in its context.).

 File "C:\Users\jerme\anaconda3\envs\so-vitts-fork\lib\site-packages\torch\distributed\distributed_c10d.py", line 998, in _new_process_group_helper
    raise RuntimeError("Distributed package doesn't have NCCL " "built in")
RuntimeError: Distributed package doesn't have NCCL built in

Any help would be greatly appreciated, and I have no problem compensating anyone who can help me solve this issue. Thx

**Comment by 34j** [comment, marked as answer]
I believe this is resolved
